dataprofiler/tests/profilers/profiler_options/test_profiler_class_options.py

```diff
@@ -1,143 +1,126 @@
 from dataprofiler.profilers.profiler_options import ProfilerOptions
 from dataprofiler.tests.profilers.profiler_options.test_base_option \
      import TestBaseOption
-
-
 class TestProfilerOptions(TestBaseOption):
-    
     option_class = ProfilerOptions
-    keys = ["structured_options"]
-    
+    keys = ["structured_options", "unstructured_options"]
     @classmethod
     def get_options(self, **params):
         options = ProfilerOptions()
         options.set(params)
         return options
-    
     def test_init(self):
         options = self.get_options()
         for key in self.keys:
             self.assertIn(key, options.properties)
-
     def test_set_helper(self):
         super().test_set_helper()
         option = self.get_options()
         optpth = self.get_options_path()
-        
```

Comment:
these changes were implemented bc this classes test didn't need to test all the options as tests in StructuredOptions should already be doing the validation for each property. Hence, we just need to reach in and validate it can propagate the results.